package.json

```diff
@@ -30,7 +30,12 @@
   "lint-staged": {
     "*.{js,ts,tsx,md,css}": "prettier --write"
   },
+  "engines": {
+    "node": ">=14.20.0",
+    "npm": ">=8.18.0"
+  },
   "devDependencies": {
+    "typescript": ">=2.8.0 || >= 3.2.0 || >= 3.3.0 || >= 3.4.0 || >= 3.5.0 || >= 3.6.0 || >= 3.7.0",
```

Comment:
Wondering what this is doing exactly, which one is it installing ?

Reply:
It is/was `typescript` @ version `v4.7.4`.. :see_no_evil: 

:heavy_check_mark: Dependencies pinned in commit: https://github.com/ledokku/ledokku/pull/421/commits/5c5bc3290d2a4f04a50e4f809e741b4dde014d1e